modules/runtime/examples/docker/cluster/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+# Example on how to use Postgres 15 with Bonita in a cluster mode
+services:
+  bonita-db:
+    image: bonitasoft/bonita-postgres:15.4
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: my-secret-pw
+    restart: always
+    # Uncomment those lines to mount a volume with your dumps
+    #volumes:
+    #  - ~/my/test/dumps:/opt/bonita/dump
+
+  # In case you have built an Application, use instead as example:
+  # my-application:
+  #  image: my-application-[my built environment]:[my aplication's semantic version]
+  bonita:
+    image: bonitasoft.jfrog.io/docker/bonita-subscription:[{bonitaVersion} for the latest maintenance version or a specific maintenance version such as {bonitaVersion}-u0]
```

Comment:
@vhemery I don't understand why did you put this sentence here 🤔

Reply:
Because I didn't understand the "pass:a" ad was mistaken by incorrect files with where "pass:" was missing (displaying "a2023.1").